webdevbroz/pomodoro-app
# Format files with prettier

## Goal
This PR is prepared by https://github.com/webdevbroz/pomodoro-app/pull/8 and the introduction of the combo ESLint + Prettier.

The goal is to deliver a good DX by formatting files automatically when saving, without having to worry about styling issues.

The formatting was done by running the following command:

"format:fix": "prettier --write --ignore-path .gitignore .

## How to test

1. In VSCode, Try to edit a file such as a tsx file from src folder by changing the spacing and anything else you want to test...
2. Save
3. The file should be formatted automatically and go back to its initial state

## Screenshots
N/A
